packages/flutter/lib/src/widgets/_html_element_view_web.dart

```diff
@@ -54,11 +56,13 @@ extension HtmlElementViewImpl on HtmlElementView {
   /// Creates the controller and kicks off its initialization.
   _HtmlElementViewController _createController(
     PlatformViewCreationParams params,
+    int viewId,
```

Comment:
`PlatformViewLink` and `PlatformViewCreationParams` are used by all platforms and I didn't want to step on their stuff. I also want to hear from @goderbauer about how other platforms are going to deal with this.

Reply:
> I didn't want to step on their stuff.

It's _our_ stuff. We own this all together. :)

Adding the viewId to PlatformViewCreationParams sounds reasonable to me. I imagine other platforms will likely also need to know what FlutterView a PlatformView is associated with. @cbracken @loic-sharma have you thought about this in the context of your Desktop Platform view work.

Reply:
One thing to note, though: A PlatformView could move from one view to another. When scrolling through this PR I didn't see anything that would update the view ID associated with the controller when this happens?

Reply:
> A PlatformView could move from one view to another.

Hm, that's an interesting case. In the web at least, since we'd need to move the actual DOM Content for platform_view (because each view has its own DOM shadow root), it'd be equivalent to destroying the view and recreating it from the factory elsewhere (in most cases).

I'm not sure how the framework/engine handles this case.

Reply:
I hadn't given this much thought I terms of desktop platform views but agreed we'll likely want this for desktop as well.